docs/chainlink-keepers/job-scheduler.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: nodes.liquid
+section: ethereum
+date: Last Modified
+title: 'Chainlink Keepers Job Scheduler'
+whatsnext:
+  {
+    'FAQs': '/docs/chainlink-keepers/faqs/',
+  }
+---
+
+
+Learn how to register a contract with the Chainlink Keepers network and automate it using the Keepers job scheduler. Before you begin, learn how to [create a Keeper-compatible contract](../compatible-contracts).
```

Comment:
Please remove this

Before you begin, learn how to [create a Keeper-compatible contract](../compatible-contracts).

When using job scheduler you don't need a Keepers-compatible contract. You do need a deployed contract with a function.

So maybe say

"When using Job Scheduler to automate your contract you just need a deployed contract on chain, it doesn't have to be Keeper-compatible"